.ghci800

```diff
@@ -43,6 +43,19 @@
 :set -Wunused-imports
 :set -Wunused-top-binds
 
+:set -Wall-missed-specialisationsg
```

Comment:
Could you motivate these settings in the PR?
What is the meaning of the appended `g`?

Reply:
g is a typo.

I normally use ghci / reply for debugging, so switch on a maximal set of warnings there.

In fact my normal preference is to go with all the warnings to begin with and switch off those that are noisy or irrelevant.  That way you get the benefit of new warnings when there is a new compiler release, for example.  These were the settings that I lighted on when looking through the code, but they are, of course, out of sync with the .cabal file and so may be out of place here.